vivisect/qt/funcgraph.py

```diff
@@ -83,16 +84,29 @@ def contextMenuEvent(self, event):
         else:
             menu = QtGui.QMenu(parent=self)
 
-        viewmenu = menu.addMenu('view   ')
-        viewmenu.addAction("Save frame to HTML", ACT(self._menuSaveToHtml))
-        #TODO: Refresh menu item :)
+        self.viewmenu = menu.addMenu('view   ')
+        self.viewmenu.addAction("Save frame to HTML", ACT(self._menuSaveToHtml))
+        self.viewmenu.addAction("Refresh", ACT(self.refresh))
```

Comment:
artifact. originally i had the menu updated in both the View and the Canvas.  i chose the callback route instead.  i'll remove that.